index.js

```diff
@@ -1,3 +1,3 @@
-const Agenda = require('./lib/agenda');
+const Agenda = require('./dist');
```

Comment:
Why is a js file being used if this is going to be ts?

Reply:
well this was the original export file.. 
but I have changed it in package.json anyways, therefore this is an old file.. i just removed it!